itou/www/apply/forms.py

```diff
@@ -1,18 +1,18 @@
 import datetime
 
 from dateutil.relativedelta import relativedelta
-
 from django import forms
 from django.conf import settings
+from django.contrib.auth import get_user_model
 from django.db.models import Q
 from django.utils import timezone
-from django.contrib.auth import get_user_model
-from django.utils.translation import gettext as _, gettext_lazy
+from django.utils.translation import gettext as _
+from django.utils.translation import gettext_lazy
```

Comment:
2 imports plutôt qu'un, pas mal isort...

Reply:
`\s`

Reply:
`combine_as_imports: If set to true - isort will combine as imports on the same line within for import statements. By default isort forces all as imports to display on their own lines.`